src/components/Wizard/CreateVmWizard/StorageTab.js

```diff
@@ -428,6 +430,7 @@ export class StorageTab extends React.Component {
               type: POSITIVE_NUMBER,
             }
           : null,
+      readValueFormatter: value => (value < 1 ? getValidK8SSize(value, this.props.units, 'Gi', false) : value),
```

Comment:
> For v2v, it's not editable already

I know, but it could be confusing for someone who looks at this in the future

> For other flows, the user is not allowed to enter floating point number to disk size, so effectively the formatting is used in the single relevant case only.

The user can have have templates with custom sizes, that would break this.

> If something, we should handle just the STORAGE_TYPE_EXTERNAL_IMPORT case, not the two mentioned.
Recently, component edit mode is disabled when renderConfig is missing. Changing that semantic would lead to further refactoring which I do not think is needed.

> IMO, sort of functional readValueFormatter is good enough. In a follow-up (out of bug-fixing scope) we can merge it with the static addendum if really needed.

You are right - I forgot about that one. 

So it probably should be  (`STORAGE_TYPE_EXTERNAL_V2V_TEMP`,  `STORAGE_TYPE_EXTERNAL_IMPORT`), because there is fixed size for both of them.

We could switch the behaviour in readValueFormatter if one of these two types is detected: 

Or rename `renderConfig` to `renderEditConfig` and create `renderValueConfig` for more generic approach. This could be a TODO.